src/core/DSPCoreDataModel.ttl

```diff
@@ -149,7 +136,7 @@ DSPCore:AlignmentFile
   owl:equivalentClass [
       a owl:Restriction ;
       owl:cardinality "1"^^xsd:nonNegativeInteger ;
-      owl:onProperty DSPCore:hasReferenceAssembly ;
+      owl:onProperty DSPCore:usedReferenceAssembly ;
```

Comment:
It looks like we're still using a `has` prefix in some places. What makes this case use a `used` prefix instead?